### PR TITLE
doc/security: add security section

### PR DESCRIPTION
**Pull request by liewegas**
- how to report an issue
- describe process for handling vulnerability
- list past issues

Signed-off-by: Sage Weil <sage@newdream.net>

### CODE REVIEW

**Comment by liewegas** [comment]
jenkins render docs

File: doc/security/index.rst [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+appreciated and will be acknowledged.
+
+
+Past vulnerabilities
+====================
+
+* 2019-03-27 `CVE-2019-3821 <https://www.cvedetails.com/cve/CVE-2019-3821/>`_
+* 2018-07-10 `CVE-2018-10861 <https://www.cvedetails.com/cve/CVE-2018-10861/>`_
+* 2018-03-19 `CVE-2018-7262 <https://www.cvedetails.com/cve/CVE-2018-7262/>`_
+* 2018-07-10 `CVE-2018-1129 <https://www.cvedetails.com/cve/CVE-2018-1129/>`_
```
Comment (idryomov) [inline doc/security/index.rst:87]: missing CVE-2018-1128

File: doc/security/index.rst [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+received a response from the Ceph security team that you can do
+so. This holds true until the public disclosure date that was agreed
+upon by the list. Thank you for improving the security of Ceph and its
+ecosystem. Your efforts and responsible disclosure are greatly
+appreciated and will be acknowledged.
+
+
+Past vulnerabilities
+====================
+
```
Comment (idryomov) [inline doc/security/index.rst:83]: missing CVE-2020-1699, CVE-2020-1700, CVE-2020-1759, CVE-2020-1760,  CVE-2020-10736, CVE-2020-10753,  CVE-2020-12059, CVE-2020-25660, CVE-2020-25677, CVE-2020-25678, CVE-2020-27781

Comment (idryomov) [inline doc/security/index.rst:83]: missing CVE-2019-10222

File: doc/security/index.rst [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+* 2019-03-27 `CVE-2019-3821 <https://www.cvedetails.com/cve/CVE-2019-3821/>`_
```
Comment (idryomov) [inline doc/security/index.rst:84]: missing CVE-2018-14662, CVE-2018-16846, CVE-2018-16889

**Review by jecluis — APPROVED** [review]

**Review by idryomov — CHANGES_REQUESTED** [review]
LGTM, but some CVEs are clearly missing so I did a quick grep.  Probably didn't get all of them as it didn't catch a couple that you listed, but better than nothing for 2020 ;)

**Review by jdurgin — APPROVED** [review]

**Review by neha-ojha — APPROVED** [review]

File: doc/security/index.rst [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
++------------+-------------------+-------------+---------------------------------------------------------+
+| 2018-07-10 | `CVE-2018-10861`_ | 8.1 High    | authenticated user can create/delete pools              |
++------------+-------------------+-------------+---------------------------------------------------------+
+| 2018-03-19 | `CVE-2018-7262`_  | 7.5 High    | malformed headers can cause RGW DoS                     |
++------------+-------------------+-------------+---------------------------------------------------------+
+| 2018-07-10 | `CVE-2018-1129`_  | 6.5 Medium  | network MITM can tamper with messages                   |
++------------+-------------------+-------------+---------------------------------------------------------+
+| 2018-07-10 | `CVE-2018-1128`_  | 7.5 High    | Cephx replay attack vulnerability                       |
++------------+-------------------+-------------+---------------------------------------------------------+
+| 2017-12-12 | `CVE-2017-12155`_ | 6.3 Medium  | ceph.client.openstack.keyring world readable            |
```
Comment (hpvyas) [inline doc/security/index.rst:127]: CVE-2017-12155 affects OpenStack product, issue was fixed in "openstack-tripleo-heat-templates/puppet-tripleo", did we ever fixed it in ceph?

File: doc/security/index.rst [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
++------------+-------------------+-------------+---------------------------------------------------------+
+| 2018-07-27 | `CVE-2017-7519`_  | 4.4 Medium  | libradosstriper format string can lead to crash         |
++------------+-------------------+-------------+---------------------------------------------------------+
+| 2015-06-08 | `CVE-2015-4053`_  |             | ceph-deploy makes admin keyring world readable          |
++------------+-------------------+-------------+---------------------------------------------------------+
+| 2015-06-16 | `CVE-2015-3010`_  |             | ceph-deploy leaves admin keyring world readable         |
++------------+-------------------+-------------+---------------------------------------------------------+
```
Comment (hpvyas) [inline doc/security/index.rst:134]: CVE-2015-3010 and CVE-2015-4053 were fixed in ceph-deploy (https://github.com/ceph/ceph-deploy) and not ceph.

File: doc/security/index.rst [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+received a response from the Ceph security team that you can do
+so. This holds true until the public disclosure date that was agreed
+upon by the list. Thank you for improving the security of Ceph and its
+ecosystem. Your efforts and responsible disclosure are greatly
+appreciated and will be acknowledged.
+
+
+Past vulnerabilities
+====================
+
```
Comment (hpvyas) [inline doc/security/index.rst:83]: Couple of more missing CVEs fixed in Ceph:

CVE-2015-5245   https://nvd.nist.gov/vuln/detail/CVE-2015-5245
CVE-2016-5009   https://nvd.nist.gov/vuln/detail/CVE-2016-5009
CVE-2016-7031   https://nvd.nist.gov/vuln/detail/CVE-2016-7031
CVE-2016-8626   https://nvd.nist.gov/vuln/detail/CVE-2016-8626
CVE-2016-9579   https://nvd.nist.gov/vuln/detail/CVE-2016-9579
CVE-2019-19337  https://nvd.nist.gov/vuln/detail/CVE-2019-19337

**Comment by idryomov** [comment]
jenkins render docs

**Pull request merged**